tests/integration/launch_container.sh

```diff
@@ -95,6 +94,7 @@ else
     --network="host" \
     ${model_path:+-v ${model_path}:/opt/ml/model:ro} \
     -v ${PWD}/logs:/opt/djl/logs \
+    -v ~/.djl.ai/:/tmp/.djl.ai \
```

Comment:
we can remove this - i don't think we want to use the cache across all tests/model

Reply:
Makes sense, removed